src/genpy/message.py

```diff
@@ -388,16 +388,11 @@ def get_printable_message_args(msg, buff=None, prefix=''):
     """
     try:
         from cStringIO import StringIO # Python 2.x
-        python3 = 0
     except ImportError:
-        from io import BytesIO # Python 3.x
-        python3 = 1
+        from io import StringIO # Python 3.x
```

Comment:
ros/ros_comm#1784 imports `BytesIO` as `StringIO`. Since here the code is using `io.BytesIO` before, should the same thing be done here?

Reply:
As far as I understand this function it should always return a string - never bytes. Therefore I think needs to be `StringIO` - independent of the Python version.